functions/src/emails.ts

```diff
@@ -390,3 +391,53 @@ export const sendNewAnswerEmail = async (
     { from }
   )
 }
+
+export const sendThreeContractsEmail = async (
+  privateUser: PrivateUser,
+  contractsToSend: Contract[]
+) => {
+  const emailType = 'weekly-trending'
+  const unsubscribeUrl = `${UNSUBSCRIBE_ENDPOINT}?id=${privateUser.id}&type=${emailType}`
+  if (!privateUser || !privateUser.email) return
```

Comment:
You should check if the privateUser is unsubscribed here.

Reply:
Well I do check that in the `weekly-markets-email` function which is what that flag is for. You can use this `sendThreeContractsEmail` function and check for your own flag.

Reply:
I suppose you'll use a different template though so I'll add it again here